post-content.js

```diff
@@ -12,6 +12,9 @@ window._wpGutenbergPost = {
 			'<section className="cover-image wp-block-cover-image" style={ { backgroundImage: \'url("https://cldup.com/GCwahb3aOb.jpg");\' } }><h2>Gutenberg Editor</h2></section>',
 			'<!-- /wp:core/cover-image -->',
 
+			'<!-- more Keep on reading! -->',
+			'<!-- noteaser -->',
```

Comment:
> `<!-- wp:core/more { "customText": "Keep on reading!", "noTeaser": true } /-->`

good question! if we do this then we _do_ have one more unique feature: the "fallback HTML" no longer carries the same rendering because WP will not appropriate handle the more tag.

as a serialized block it will only work if we have a block registered.

I'm not sure how I feel, but I have a commit ready to serialize the normal more tag